apps/iOS-WKWebView-sample/README.md

```diff
@@ -0,0 +1,27 @@
+# Amazon Chime SDK WKWebView Sample
+
+## Summary
+
+This sample shows how to run a WebRTC based meeting application inside of a iOS WKWebView. Prior to iOS 14.3, WebKit did not support WebRTC, meaning WKWebViews were not able to access WebRTC APIs. iOS 14.3 released support for WebRTC in WebKit - allowing devlopers to run WebRTC-based applications in a WKWebView. The sample itself is a bare-bones iOS app that loads a WKWebView, which then navigates to a specified URL.
```

Comment:
- `a iOS WKWebView` => `an iOS WKWebView`
- How about `WebKit did not` => `WKWebView did not` (or `iOS WebKit`) since WebKit itself supports a part of WebRTC since iOS 11 (https://webkit.org/blog/7726/announcing-webrtc-and-media-capture)?
- `devlopers` => `developers`